keras_cv/layers/aspp.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any
+from typing import List
+from typing import Mapping
+
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class ASPP(tf.keras.layers.Layer):
```

Comment:
"ASPP" as a name is not a broadly understood abbreviation. We should use a fully spelled out name.

Reply:
I can change it to  "AtrousSpatialPyramidPooling" which is the full name of ASPP.

Reply:
ASPP name is very popular

https://pytorch.org/vision/stable/_modules/torchvision/models/segmentation/deeplabv3.html

Reply:
Actually ASPP name sounds good to me as well, but I definitely understand the confusion, and also the inconsistency with general Keras naming....

Reply:
> inconsistency with general Keras naming....
Yes this is important especially as we already accumulated some naming debt in Keras-cv:
https://github.com/keras-team/keras-cv/issues/826

Reply:
ASPP is popular but can really mean a lot of things. Googling ASPP results in most links not being about Atrous Spatial Pyramid Pooling, though, the images are mainly related to DeepLabV3+ at least. Maintaining naming conventions might be more important than the popularity of the term usage, though